components/build/kustomization.yaml

```diff
@@ -3,7 +3,6 @@ resources:
 - tekton-results/
 - build-templates/
 - pipelines-as-code/
-- tekton-chains/
```

Comment:
I think it makes sense to move this from `build` to `enterprise-contract`. I'm just worried what will happen during an upgrade. Will we have two Applications trying to manage the same resources?

Reply:
There should be only one Application, well -- two: `enterprise-contract` and (in some cases) `rekor`. ArgoCD should make sure that the definitions that for Tekton Chains that were loaded from the `build` application are replaced with the configuration now loaded from `enterprise-contract` application.